src/Microsoft.ML.Ensemble/Trainer/Multiclass/MulticlassDataPartitionEnsembleTrainer.cs

```diff
@@ -66,7 +66,7 @@ public Arguments()
                             // estimator, as opposed to a regular trainer.
                             var trainerEstimator = new LogisticRegressionMulticlassClassificationTrainer(env, LabelColumnName, FeatureColumnName);
                             return TrainerUtils.MapTrainerEstimatorToTrainer<LogisticRegressionMulticlassClassificationTrainer,
-                                MulticlassLogisticRegressionModelParameters, MulticlassLogisticRegressionModelParameters>(env, trainerEstimator);
+                                LogisticRegressionMulticlassModelParameters, LogisticRegressionMulticlassModelParameters>(env, trainerEstimator);
```

Comment:
We can NOT swap them. Logistic regression is never a multiclass classification model while multi-class logistic regression is an alternative name of `[multinomial logistic regression](https://en.wikipedia.org/wiki/Multinomial_logistic_regression)`

Reply:
One alternative name we can use is SoftmaxRegression (also mentioned in wikipedia link above).

While refactoring, could you fix the name of the trainer estimator for this as well. 

I will revert this, and wait for your refactoring PR. Sounds good ?

---
In reply to: [265806976](https://github.com/dotnet/machinelearning/pull/2968#discussion_r265806976) [](ancestors = 265806976)